src/Installers/Windows/WindowsHostingBundle/WindowsHostingBundle.wixproj

```diff
@@ -53,20 +54,7 @@
       <Private>True</Private>
       <DoNotHarvest>true</DoNotHarvest>
     </ProjectReference>
-    <ProjectReference Include="..\SharedFrameworkLib\SharedFrameworkLib.wixproj">
-      <SetPlatform>Platform=x86</SetPlatform>
-      <Name>SharedFrameworkLib_x86</Name>
-      <Project>{5244BC49-2568-4701-80A6-EAB8950AB5FA}</Project>
-      <Private>True</Private>
-      <DoNotHarvest>true</DoNotHarvest>
-    </ProjectReference>
-    <ProjectReference Include="..\SharedFrameworkLib\SharedFrameworkLib.wixproj">
-      <SetPlatform>Platform=x64</SetPlatform>
-      <Name>SharedFrameworkLib_x64</Name>
-      <Project>{5244BC49-2568-4701-80A6-EAB8950AB5FA}</Project>
-      <Private>True</Private>
-      <DoNotHarvest>true</DoNotHarvest>
-    </ProjectReference>
+    <ProjectReference Include="..\SharedFrameworkBundle\SharedFrameworkBundle.wixproj" />
```

Comment:
Suggest this reference should have `ReferenceOutputAssembly="false"` and `Private="True"`

Reply:
Done